src/webgpu/api/operation/storage_texture/read_write.spec.ts

```diff
@@ -318,7 +318,9 @@ g.test('basic')
   .fn(t => {
     const { format, shaderStage, textureDimension, depthOrArrayLayers } = t.params;
 
-    const kWidth = 16;
+    // In compatibility mode the maximum minimum invocations is 128 vs non-compat which is 256
```

Comment:
`maxComputeInvocationsPerWorkgroup`?

Q: where do we see these limits? Do we have a compat page for https://gpuweb.github.io/gpuweb now?

Reply:
they are here: https://github.com/gpuweb/gpuweb/pull/4397/files

Reply:
Merged so now it's here: https://github.com/gpuweb/gpuweb/blob/main/proposals/compatibility-mode.md#10-lower-limits

Reply:
Prefer. (maximum minimum sounds confusing)

```suggestion
    // In compatibility mode the lowest maxComputeInvocationsPerWorkgroup is 128 vs non-compat which is 256
```